tests/PnG.tex

```diff
@@ -0,0 +1,118 @@
+\section{Procter \& Gamble Challenge [DSPL \& OPL]}
+
+\definecolor{tpblue}{HTML}{182085}%
+\definecolor{tporange}{HTML}{FD5B10}%
+\newcommand{\TidePod}{%
+	\textit{%
+		\fontfamily{pvh}\selectfont%
+		P\&G
+		\fontfamily{phv}\selectfont%
+		\textcolor{tpblue}{Tide}%
+		\textcolor{tporange}{\textbf{\upshape POD}}%
+	}%
+}
+\newcommand{\openpart}{%
+All teams are allowed to participate and compete in the \textit{Procter \& Gamble Challenge} regardless of whether they advanced to the Stage II or not, and get the award.
+}
+
+The robot has to remove all dishes from a table (presumably after dinner) and place them into the dishwasher.
+
+\subsection{Open Participation}
+\openpart
+
+\subsection{Focus}
+This test focuses on object perception, manipulation, and planning.
+
+\subsection{Setup}
+\begin{itemize}
+	\item \textbf{Location:} This test takes place in the arena. A dining table is located close to the dishwasher.
+	\item \textbf{Dishwasher:} The dishwasher is near the table, preferably located in the same room. The dish washer is open and with all racks out.
+	\item \textbf{Tray:} A plastic tray is located either on top of the dishwasher, or onto one of its racks. The tray may have tableware and cutlery placed inside already.
+	\item \textbf{Table setting:} The table has several objects disposed in a typical setting for a meal for one person. These objects include tableware (e.g. place mats, napkins, dishes, glasses), and silverware (e.g. forks, spoons, knives).
+	\item \textbf{Spot:} There is a spot on the table next to the table setting.
+\end{itemize}
+
+\subsection{Task}
+	\begin{enumerate}
+		\item \textbf{Entering the arena:} The robot enters the arena and navigates to the designated location.
+		\item \textbf{Fetching command:} The operator requests the robot to clean up the table.
+		\item \textbf{Clean the table:} The robot takes all the tableware and cutlery to either the dishwasher or to the tray, as instructed (team's choice).
+		\item \textbf{Filling the dishwasher:} If the robot placed the objects into the tray, it must proceed to put the tray onto one of the dishwasher racks.
+		\item \textbf{Place the \TidePod:} The robot places the  \TidePod~into the dishwasher, preferrably inside the tab compartment.
```

Comment:
Tidepod -> Cascade Pod

Reply:
Why Cascade Pod? I understood TidePOD was P&G family

![image](https://user-images.githubusercontent.com/8583949/37732872-98f19244-2d46-11e8-8581-7350dfe7bde6.png)

Reply:
Fabio stated so in his email.

Reply:
Fair enough!